src/core/reducers/search.js

```diff
@@ -4,13 +4,20 @@ import {
 } from 'core/constants';
 import { createInternalAddon } from 'core/reducers/addons';
 
+
+const ABORT_SEARCH = 'ABORT_SEARCH';
```

Comment:
for consistency with these other action names, it should probably be `SEARCH_ABORTED`.